### PR TITLE
TRAND Validation/Payload Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15582

*Description of changes:*
* Implements Validations for Transfer to Surviving Joint Tenant
* Updated Permissions on Transfer options
* Misc UX UI adjustments for Death Certificate

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrTransfers/TransferType.vue [hunk 1]
```diff
@@ -213,10 +221,34 @@ export default defineComponent({
...
+          localState.transferTypeRules = customRules(
+            required('Select transfer type'),
+            [
+              () => isJointTenancyStructure.value ||
+              'An owner group with joint tenancy is required for this transfer type'
+            ]
+          )
+          break
+      }
+    })
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/TransferType.vue:244]: Similar to the other rule switch cases, we can expand here as we go depending on transfer type.

File: ppr-ui/src/views/newMhrRegistration/HomeOwners.vue [hunk 2]
```diff
@@ -414,7 +414,10 @@ export default defineComponent({
       })
     })
 
-    const hideShowRemovedOwners = (): void => {
+    const hideShowRemovedOwners = (forceShow: boolean = false): void => {
+      // Override Toggle to force show
+      if (forceShow) localState.hideRemovedOwners = true
+
       localState.hideRemovedOwners = !localState.hideRemovedOwners
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/HomeOwners.vue:421]: UX requirement to show the embedded Death Certificate form if it is invalid AND the user has hidden the removed/deceased owners.

**Review by owillborn — APPROVED** [review]
Lots of work here, looks bueno

**Review by dimak1 — APPROVED** [review]
Looks good to me!

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Pull request merged**